pyprep/find_noisy_channels.py

```diff
@@ -183,16 +195,26 @@ def find_all_bads(self, ransac=True, channel_wise=False, max_chunk_size=None):
             effect. Defaults to ``None``.
 
         """
-        self.find_bad_by_nan_flat()
```

Comment:
Yep! Originally I left it because I thought it'd be trivially fast to re-run, but in my real-world testing with 32ch / 1000 Hz files it still takes a second or two to run the method each time.

Reply:
okay, makes sense, but perhaps the comment could be replaced by a comment about that (instead of deleted)

Reply:
Done in the latest commit.